src/pretty.js

```diff
@@ -0,0 +1,52 @@
+import chalk from 'chalk';
+
+function stats(articles, searchQuery, options) {
+  return chalk.magenta(
+    `Found ${articles.length} articles` +
+    (searchQuery ? ` for "${searchQuery}"` : '') +
+    (options.strict ? ' with strict filtering' : '') + 
+    (options.reverse ? ' in reversed order' : '') +
+    '.'
+  );
+}
+
+function highlight(text, term) {
+  const regex = new RegExp('(' + term + ')', 'gi');
+  
+  return text.replace(regex, chalk.magenta('$1'));
+}
+
+export function prettyArticle(article, searchQuery = '', options) {
+  let title = article.title;
+  let abstract = article.abstract;
+  
```

Comment:
For download case, `article` attributes still have the 'preprint_' prefixes, so it passes text=undefined to the `highlight` function. Anyways, the downloads will change with my updates so I wouldn't worry about this case right now.

Reply:
Yeah, I figured the normalisation should probably happen on the download level.  Otherwise, it's annoying to pass around files when you just want to search etc.

Reply:
We could also remove the pretty printing on the download completely.

The search could just print everything if your search query is empty, so you could use search if you wanted to just pretty format a JSON file.